llvm/lib/Target/AArch64/SMEInstrFormats.td

```diff
@@ -2180,10 +2191,11 @@ multiclass sme2_int_mla_long_array_vg2_single<string mnemonic, bits<2> op, SDPat
 
 multiclass sme2_fp_mla_long_array_vg4_single<string mnemonic, bits<3> op, MatrixOperand matrix_ty,
                                              RegisterOperand multi_vector_ty, ZPRRegOp vector_ty,
-                                             ValueType zpr_ty, SDPatternOperator intrinsic> {
+                                             ValueType zpr_ty, SDPatternOperator intrinsic, list<Register> uses=[]> {
```

Comment:
Instead of adding uses as extra argument, I think it would be better to use:
`let Uses = [FPMR, FPCR] in {
...intrinsics
}`